packages/core/src/components/tooltip/_tooltip.scss

```diff
@@ -24,7 +24,7 @@
   // lightweight scale for tooltips
   @include react-transition(
     "pt-popover",
-    (transform: scale(0.8) scale(1)),
+    (transform: scale(1) scale(1), opacity: 0 1, filter: blur($pt-grid-size) blur(0)),
```

Comment:
add note about canceling `transform` transition from popover, so we don't wonder why these are the same scales